src/otx/core/model/entity/base.py

```diff
@@ -345,7 +356,13 @@ def __init__(
 
         tile_enabled = False
         with contextlib.suppress(RuntimeError):
-            tile_enabled = "tile_size" in self.model.inference_adapter.get_rt_info(["model_info"]).astype(dict)
+            if isinstance(self.model, Model):
+                tile_enabled = "tile_size" in self.model.inference_adapter.get_rt_info(["model_info"]).astype(dict)
+            else:
```

Comment:
I just set if-else because `self.model` for visual prompting is `dict`, not `Model`.
Actually, tiling for visual prompting is not supported yet, but it can be expanded.
I removed it at https://github.com/openvinotoolkit/training_extensions/pull/2955/commits/d9335dec60ff9a5e096b1317a61042f10df00208.